Nice (libnice) port

iraklisk opened the pull request:
Nice (libnice) port for version 0.1.13 based on cmake.

iraklisk committed:
libnice port

iraklisk committed:
Remove auto generated comments

msftclas commented:
Hi __@iraklisk__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, MSBOT;

ras0219-msft commented:
Awesome job on your first port!

What auto generator did you use, by the way? It looks pretty effective, so I'd like to add it to the examples document or even the "create" command.

iraklisk commented:
The specific project includes a .vcproj. 
I upgraded the project to .vcxproj and then i used [this powershell script](https://github.com/nberserk/common/blob/master/vcproj2cmake.ps1) in order to avoid typing the header and source files.
Then i fixed manualy the dependencies, the compiler flags for the warnings , the targets and the include files.

I also tried to build the specific project with msbuild after i converted to vcxproj and upgrade the toolchain, but I ran into a problem with  /p:VCPkgLocalAppDataDisabled=true which has no override in vcpkg_build_msbuild.cmake.

ras0219-msft commented:
Thanks!
